cmake/SetupPackages.cmake

```diff
@@ -29,7 +29,7 @@ if (RAJA_ENABLE_TBB)
   endif()
 endif ()
 
-if (RAJA_ENABLE_CUDA OR RAJA_ENABLE_EXTERNAL_CUB)
+if (RAJA_ENABLE_CUDA AND RAJA_ENABLE_EXTERNAL_CUB)
```

Comment:
We need to check that this will still pick up the submodule if that's what one wants.

Reply:
For me it worked. Before it wasn't installing the cub files and now it is.

Reply:
Great. I'm testing locally now. I added the folks who touched that file last for review. we'll discuss this at the RAJA meeting next Tuesday (Monday is a lab holiday).